### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by nongdenchet** [DangFiztssi/Assignment-1-Flicks#3]
Hi could you please attach the GIF of your application

**Comment by ngdang95** [comment]
@nongdenchet yes, I attached the GIF in README file, specifically in here
[demo Flicks](https://github.com/DangFiztssi/Assignment-1-Flicks/blob/master/demo_flicks.gif)
Hope you like it :)

**Comment by coderschoolreview** [comment]
### Thank you for your submission
- :+1: Thank you, this is a great work.
- The point of this homework was to explore some basic Android components with a RESTful API. You learned about RecyclerView, custom Adapter, navigation through application and basic networking.

### Good stuffs
- I see that you learned custom Adapter by using `RecyclerView.Adapter` and `ViewHolder` pattern.
- Nice work implementing `Presenter` and seperate `interface`.
- So far, you known how to use `Retrofit` for networking stuff.
- Nice job using `ButterKnife` to bind view.
- Nice job implementing `Picasso` to load image. In addition, you can `transform` image.
- Great to see you implement `YouTubePlayerView`.
- Your code convention and project structure are really good. Try to read more at [android-guidelines](https://github.com/ribot/android-guidelines/blob/master/project_and_code_guidelines.md).

### Suggestions
- Basically, the main activity is better if you implement the search function.
- I wish that you write down some challenges after each assignment, and how can you pass over all of them :heart_eyes: :muscle:
